lang/en/block_progress.php

```diff
@@ -251,5 +253,3 @@
 // For My home page.
 $string['no_blocks'] = "No Progress Bar blocks are set up for your courses.";
 $string['no_courses'] = "You are not enrolled in any courses. Only bars from enrolled courses will be shown.";
-
-$string['privacy:metadata'] = "Progress block does not store any personal data.";
```

Comment:
I'm not sure why we delete this

Reply:
I've added this item back in, it must have slipped while resolving conflicts: https://github.com/catalyst/moodle-block_progress/pull/1/commits/6eff21e979c9cc892ef99ed1944d9e297c1769b3 .